File: JAVA/NumberGuessingApp.java
import java.util.Random;
import java.util.Scanner;
public class NumberGuessingApp {
    public static void main(String[] args) {
            Scanner inputCollector = new Scanner(System.in);
            Random rand = new Random();
            int rightNumber = rand.nextInt(1 , 100);
        
            int num = 0;
//        for (int count = 1; count <= 1; count++){
            System.out.print("Guess your number :  ");
            int number = inputCollector.nextInt();

                if (number > 0){
                  if (number < 101) {
                    if (number == rightNumber) {
                        num++;
                      System.out.println(rightNumber);
                    }
                    else {
                         for (int counter = 1; counter <= 5; counter++)    {
                            if (number > rightNumber){
                                    System.out.println("It's too high!!'");
                            }
                            else{
                                    System.out.println("It's too low!!'");
                            }
                            System.out.print("Guess your number :  ");
                                number = inputCollector.nextInt();
                                if (number == rightNumber) {
                                   System.out.println(rightNumber);
                                    num++;
                                     break;
                               }
                                }
                            }       
                        }

                }
                else{
                            System.out.println("Wrong input!!! , input number within range 1 to 100");                  
                    }
                                   System.out.println(rightNumber);
//        }
    }
}

File: PYTHON/number_guessing_app.py
import random

number = int(input("Guess the number :  "))
right_number = random.randrange(1 , 101)

if (number <= 100) and (number >= 1):
    j = 0
    for count in range(1 , 2):
        if (number != right_number):
            for counter in range(1 , 5):
                if (number > right_number):
                    print("Number is higher!!!")
                else:
                    print("Number is lower!!!")
                number = int(input("Guess the number :  "))
                if (number <= 100) and (number >= 1):
                    if (number == right_number):
                        break
                else:
                    print("Wrong input!! , enter a number within the range of 1 to 100!!")  
                    for count in range(1 , 1000):
                        number = int(input("Guess the number :  "))
                        print("Wrong input!! , enter a number within the range of 1 to 100!!")  
                        if (number <= 100) and (number >= 1):
                            if (number > right_number):
                                print("Number is higher!!!")
                            else:
                                print("Number is lower!!!")
                            break
        else:
            j = j + 1
            if (j == 1):
                print("Legendary!!")
            elif (j == 2):
                print("Excellent!!")
            elif (j == 3):
                print("Good")
            elif (j == 4):
                print("Good")
            elif (j == 5):
                print("close!")
            elif (j == 0):
                print("No win , Better Luck")
    print("Correct number is " , right_number)

else:
    print("Wrong input!! , enter a number within the range of 1 to 100!!")




